### PR TITLE
Add SOLO RPW spectrogram support

### LINKED ISSUE #55: Add support for SolarOrbiter Radio and Plasma Waves data
### Description

* Use the SOAR archive fido client for each and download
* Add spectrogram class

### PR DESCRIPTION
**Pull request by samaloney**
### Description

Fixes #55 adding support for some SOLO//RPW data 

```python
import sunpy_soar
from astropy.time import Time
from astropy.visualization import ImageNormalize, PercentileInterval
from sunpy_soar.attrs import Identifier
from sunpy.net import Fido, attrs as a
from matplotlib import pyplot as plt

from radiospectra.spectrogram import Spectrogram

query = Fido.search(a.Time('2020-07-11', '2020-07-11 23:59'),
                    a.Instrument('RPW'), a.Level(2), Identifier('RPW-HFR-SURV'))
downloaded = Fido.fetch(query[0])
spec = Spectrogram(downloaded[0])

fig, ax = plt.subplots()
norm = ImageNormalize(spec[0].data, interval=PercentileInterval(97.5))
spec[0].plot(norm=norm, axes=ax)
ax.set_xlim([Time('2020-07-11 2:00').datetime, Time('2020-07-11 3:00').datetime])
ax.semilogy()
```
solo_rpw

### CODE REVIEW

**Comment by samaloney** [comment]
This code doesn't work on all RPW spectra need to test if any of acg1 or acg2 and use one or both accordingly

File: radiospectra/spectrogram2/spectrogram.py [hunk 1]
```diff
@@ -671,6 +674,79 @@ def _read_cdf(file):
...
+                freqs = []
+                for cfreq in freq_uniq:
+                    search = np.argwhere((all_freqs == cfreq) & (sensors[:, sensor_num] == sensor_value) & (acg != 0))
+                    if search.size > 0:
+                        slices.append(acg[search])
+                        times.append(all_times[search])
+                        freqs.append(cfreq)
+                return slices, times, freqs
+
+            sensor_unique = np.unique(sensor)
```
Comment (hayesla) [inline radiospectra/spectrogram2/spectrogram.py:705]: is this the correct thing to be indexing? 

Is the idea to find CH1 and CH2 here?

**Comment by nabobalis** [comment]
Probably want more source tests?

**Comment by nabobalis** [comment]
Online fail is very real.

File: radiospectra/net/sources/tests/test_stereo_client.py [hunk 2]
```diff
@@ -19,6 +19,7 @@ def client():
     return SWAVESClient()
 
 
+@pytest.mark.skip("Upstream data server removed")
```
Comment (nabobalis) [inline radiospectra/net/sources/tests/test_stereo_client.py:22]: Will it come back?

Comment (samaloney) [reply radiospectra/net/sources/tests/test_stereo_client.py:22]: Nope the job of processing/hosting has moved to a different group/institute and the data is now only available on the [CDPP](https://cdpp-archive.cnes.fr) and I'm not sure how to search that via an API also I think the files are in CDF format.

Comment (nabobalis) [reply radiospectra/net/sources/tests/test_stereo_client.py:22]: So we should remove these tests and the client for it?

Comment (samaloney) [reply radiospectra/net/sources/tests/test_stereo_client.py:22]: Yea I need to check where the data can be obtained and in what formats so want to leave the code until I figure that out, only skipped the tests for the CI really.

**Pull request merged**